internal/services/compute/managed_disk_resource.go

```diff
@@ -187,6 +187,17 @@ func resourceManagedDisk() *pluginsdk.Resource {
 				ValidateFunc: validation.IntBetween(2, 10),
 			},
 
+			"hyper_v_generation": {
```

Comment:
lets leave version on it and maybe add a note to that other file to rename it to be more clear - thanks!

Reply:
sure, will use hyper_v_generation_version instead and rename the property in other files

Reply:
generation and version mean essentially the same thing in this context - but this is known as the Generation of HyperV, rather than Version - so we should use `generation` and not `version` here